submissions/ighawosaOmoma/Javascript/conditionals.js

```diff
@@ -0,0 +1,17 @@
+var positiveDifference= 123-7
+// Array.from(positiveDifferenc).forEach(function(num){
+//     console.log(num)
+// })
+for(var i=1; i<positiveDifference; i++){
+    if(i % 3 ===0 && i % 5 ===0){
+        document.write('Software Developer <br>')
+    }
+    else if (i % 3 ===0){
+        document.write('Software <br>')
+    }
+    else if(i % 5 ===0){
+        document.write('Developer <br>')
```

Comment:
This line is not well concatenated. Please , insert a plus sign in between Developer and br just like you did in line 15 and put the quotes appropriately. Do the same for lines 7 & 10.  Run your code and attach the a Screenshot of your output to this PR. Kindly tag me on Slack @Bukky, once you are done.